api/audio_codecs/opus/audio_decoder_opus.cc

```diff
@@ -44,6 +44,9 @@ absl::optional<AudioDecoderOpus::Config> AudioDecoderOpus::SdpToConfig(
         return absl::nullopt;  // Bad stereo parameter.
       }
     }
+#ifdef OWT_CLOUD_GAMING
+    return 2;
```

Comment:
I remember in close source OWT we set the bandwidth of opus to 256k. Will you also configure it there?

Reply:
Change for default bandwidth has been picked earlier. https://github.com/open-webrtc-toolkit/owt-deps-webrtc/blob/bd94d8f2af05943925636a82eb474f17ea4e439d/modules/audio_coding/codecs/opus/audio_encoder_opus.cc#L98